modules/monitoring-configuring-remote-write.adoc

```diff
@@ -0,0 +1,176 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+[id="configuring_remote_write_storage_{context}"]
+= Configuring remote write storage
+
+You can configure remote write to enable Prometheus to send ingested metrics to remote systems for storage. 
+Doing so has no impact on how or for how long Prometheus stores metrics using local storage.
+
+.Prerequisites
+
+* *If you are configuring core OKD monitoring components:*
+** You have access to the cluster as a user with the `cluster-admin` role.
+** You have created the `cluster-monitoring-config` `ConfigMap` object.
+* *If you are configuring components that monitor user-defined projects:*
+** You have access to the cluster as a user with the `cluster-admin` role or as a user with the `user-workload-monitoring-config-edit` role in the `openshift-user-workload-monitoring` project.
+** You have created the `user-workload-monitoring-config` `ConfigMap` object.
+* You have installed the OpenShift CLI (`oc`).
```

Comment:
@rolfedh do we list this on other pre-requsites? Is this implied?

Reply:
Yes, this line is used by 74 instances of other files throughout the documentation.

Reply:
I'm leaving the wording as-is to stay consistent with the rest of the docs.